SonarQube.Client/Api/V5_10/GetIssuesRequest.cs

```diff
@@ -88,6 +88,6 @@ private static IEnumerable<T> ReadFromProtobufStream<T>(Stream stream, MessagePa
 
         private static SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
             new SonarQubeIssue(issue.Path, issue.Checksum, issue.Line, issue.Msg, issue.ModuleKey, issue.RuleKey,
-                issue.Status == "RESOLVED");
+                issue.Status == "RESOLVED", flows: null);
```

Comment:
I don't known why the returned status is hard-coded to `RESOLVED` - possibly because we don't use it at the moment. I'll create a separate ticket to look at it.